charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -41,7 +41,7 @@ spec:
             - /run/spire/config/server.conf
           ports:
             - name: grpc
-              containerPort: 8081
+              containerPort: {{ .Values.service.port }}
```

Comment:
doesn't have to necesarrely be the same port as mapped in the service.

Reply:
If this wasn't an oversight/bug, I will revert this change.